CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ _None_
 
 ### Internal Changes
 
-_None_
+Adopts Buildkite
```

Comment:
```suggestion
- Adopt Buildkite
```
I think our past wording was more either infinitive or using the 1st person "[We now] adopt" rather than speaking for the tool as "[It] adopts" 🤷‍♂️

Also adding bullet point for consistency with those sections being lists, even if only one item in it so far.